newIDE/app/src/ProjectsStorage/LocalFileStorageProvider/LocalProjectWriter.js

```diff
@@ -24,6 +24,32 @@ export const splittedProjectFolderNames = [
   'eventsFunctionsExtensions',
 ];
 
+const deleteExistingFilesFromDirs = (projectPath: string) => {
+  const entries = fs.readdirSync(projectPath);
+  return new Promise((resolve, reject) => {
+    //Project file
+    if (entries.length === 1) resolve();
+
+    //If multiFile enabled in settings and directories exist!
```

Comment:
You do not actually check if multifile is enabled in the project's settings. You should use `project.isFolderProject()` for that.

Reply:
To make it one step clearer by removing the indent, you could do this:

```js
if (!project.isFolderProject()) return;

const entries = fs.readdirSync(projectPath);
...
```

Also, I'm not sure this comment is useful anymore, your code is quite clear.